### ISSUE TITLE
manual/Highlights22.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#857]
- https://github.com/playframework-ja/translation-project/tree/2.3.x/manual/Highlights22.md
- diffs between [2.2.0/manual/Highlights22.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/Highlights22.md) and [2.3.x/manual/Highlights22.md](https://github.com/playframework-ja/translation-project/tree/2.3.x/manual/Highlights22.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/Highlights22.md 2013-09-19 10:11:22.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/Highlights22.md    2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 # What's new in Play 2.2

 ## New results structure for Java and Scala
@@ -13,7 +14,7 @@
 '''scala
 def index = Action.async {
   val foo: Future[Foo] = getFoo()
-  foo.map(f => Ok(foo))
+  foo.map(f => Ok(f))
 }
 '''

@@ -91,3 +92,11 @@
 ## Built in gzip support

 Play now has built in support for gzipping all responses.  For information on how to enable this, see [[Configuring gzip encoding|GzipEncoding]].
+
+## Documentation JAR
+
+Play's distribution now stores its documentation in a JAR file rather than in a directory. A JAR file provides better support for tooling.
+
+Just like in Play 2.1, you can view the documentation when you [[run your Play application in development mode|PlayConsole]] by visiting the special [`/@documentation`](http://localhost:9000/@documentation) address.
+
+If you want to access the raw files, they can now be found in the `play-docs` JAR file contained in the distribution.

```

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1081
